source/release-notes/release_3_13_0.rst

```diff
@@ -23,34 +24,34 @@ Wazuh core
 - Added a timeout to the updates of the vulnerability detector's feeds to prevent hangings.
 - Added option for the JSON decoder to choose the treatment of array structures.
 - Added ``mode`` value (real-time, Who-data, or scheduled) as a dynamic field in FIM alerts. 
-- Added a configurable maximum limit of files to be monitored by FIM.
-- New integration for pull logs from Google Cloud Pub/Sub.
-- Added support for MITRE ATT&CK knowledge base.
+- Added a field to configure the maximum files to be monitored by the FIM.
```

Comment:
By the FIM? FIM module sounds better, right?

Reply:
Agree